lib/ts/MatcherUtils.cc

```diff
@@ -70,30 +72,50 @@ readIntoBuffer(const char *file_path, const char *module_name, int *read_size_pt
     return nullptr;
   }
 
-  if (file_info.st_size < 0) {
+  file_size = file_info.st_size; // number of bytes in file
+
+  if (file_size < 0) {
     Error("%s Can not get correct file size for %s file : %" PRId64 "", module_name, file_path, (int64_t)file_info.st_size);
     close(fd);
     return nullptr;
   }
+
+  ink_assert(file_size >= 0);
+  ink_assert(file_size == file_info.st_size);
```

Comment:
This isn't necessary since you are doing an assignment right above.

Reply:
removed.